api/v1/openapi.yaml

```diff
@@ -1235,41 +1235,55 @@ definitions:
         type: object
         additionalProperties:
           type: string
-  DaemonConfigurationResponse:
+  DaemonConfiguration:
+    description: |
+      Response to a daemon configuration request.
+    type: object
+    properties:
+      spec:
+        description: Changeable configuration
+        "$ref": "#/definitions/DaemonConfigurationSpec"
+      status:
+        description: Changeable configuration
```

Comment:
This is inconsistent with the description of the `DaemonConfigurationStatus` type below, which states that this contains also immutable configuration settings.